decomposed/campaign/The Dream-Eaters/TheDream-Eaters.a16a1a/2-ATheSearchForKadath.7b07ca/SearchforKadathHelper.c91b58.ttslua

```diff
@@ -164,6 +161,7 @@ function setup(region, color)
   Wait.time(function() mythosAreaApi.reshuffleEncounterDeck() end, 0.1)
   Wait.time(function() cleanZone() end, 1)
   Wait.time(function() placeLocations(region) end, 2)
+  Wait.time(function() mythosAreaApi.getEncounterDeck().shuffle() end, 0.1)
```

Comment:
Looks like we are shuffling twice?

Reply:
Those are two different functions. 161 shuffles in anything that was in the discard. cleanZone() and placeLocations potentially adds in more cards to the encounter deck (or takes them out), 164 is just a fail-safe to shuffle the encounter deck in case it wasn't done already.

Reply:
That looks a bit messy, but I'm fine with that for now